deckhouse-controller/crds/module-source.yaml

```diff
@@ -95,6 +95,9 @@ spec:
                         type: string
                       error:
                         type: string
+                configChecksum:
```

Comment:
Usually, status is for humans and machines, not for the controllers. If you want to keep the state between configuration changes, the annotation is the best way to do this. This is how controllers operate in Kubernetes.

Reply:
accepted and rewrote to use an annotations instead of .status